controllers/chat/group/groupAddMembers.js

```diff
@@ -68,21 +67,28 @@ const groupAddMembers = async (req, res) => {
       const textTargetUser = `${ownUser.username} added you to this group`;
       const textDefaultUser = `${ownUser.username} added ${targetUserModel.username} to this group`;
 
-      await channelToAdd.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        other_system_user: member,
-        better_type: 'add_member_to_group',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: [member]
-      });
+      await channelToAdd.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          other_system_user: member,
+          better_type: 'add_member_to_group',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: [member]
+        },
+        {
+          skip_push: true
+        }
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessage` function now includes an additional object with a `skip_push: true` property. This change suggests that the action of adding a member to a group will now skip pushing notifications. However, immediately after this, a notification is manually sent using `BetterSocialCore.fcmToken.sendGroupChatNotification(member, textTargetUser)`. 

This seems contradictory and could lead to confusion or unexpected behavior. If the intention is to control when and how notifications are sent, it would be better to make this clear in the code or comments.

If the `skip_push` option is intended to prevent automatic notifications but still allow manual ones, then this code is fine as it is. However, if the `skip_push` option is supposed to prevent all notifications, then the manual notification should not be sent.

Please ensure that the usage of `skip_push` and `sendGroupChatNotification` aligns with your intended functionality.

```diff
      await channelToAdd.sendMessage(
        {
          text: textDefaultUser,
          own_text: textOwnUser,
          other_text: textTargetUser,
          other_system_user: member,
          better_type: 'add_member_to_group',
          type: 'system',
          user_id: userId,
          only_to_user_show: userId,
          disable_to_user: false,
          is_from_prepopulated: true,
          system_user: userId,
          isSystem: true,
          members: [member]
        },
        {
          skip_push: true
        }
      );

+      // Ensure that this manual notification sending aligns with the intended use of skip_push.
      await BetterSocialCore.fcmToken.sendGroupChatNotification(member, textTargetUser);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->